src/Symfony/Component/Translation/TranslatorBagInterface.php

```diff
@@ -21,11 +21,11 @@
     /**
      * Gets the catalogue by locale.
      *
-     * @param string|null $locale The locale or null to use the default
+     * @param string $locale The locale
      *
      * @throws \InvalidArgumentException If the locale contains invalid characters
      *
      * @return MessageCatalogueInterface
      */
-    public function getCatalogue($locale = null);
+    public function getCatalogue($locale);
```

Comment:
Apart from whether PHP would allow this or not – do you have a reasonable (default) locale in all places that call this?